### DISCUSSION TITLE
Got a mail saying suspicious login and I am unable get reset password link

### DISCUSSION THREAD

**Discussion by ghost** [community/community#140782] [Other Feature Feedback, Questions, & Ideas] [answered]
I am unable to get password reset link to my mail ID I tried in many ways and checked even in bin folder but no use. The account is linked to my  institute mail ID. I am trying to recover my account since a month no use please help me to recover.
Account : SaiTejaMantha03
Mail ID:[Redacted by moderator due to PII]
Please help in recovering..

**Comment by queenofcorgis** [comment, marked as answer]
Hi @SaiMantha03 

Thank you for your patience. The community is unable to assist with nor escalate your account recovery. Please use [this doc](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials) for help with your recovery. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
